src/filtrering/filtrering-status/filtrering-status.tsx

```diff
@@ -54,25 +80,29 @@ export function FiltreringStatus(props: FiltreringStatusProps) {
         }
     }
 
-    const statusTall = useStatusTallSelector();
     const erVedtaksStotteFeatureTogglePa = useFeatureSelector()(VEDTAKSTOTTE);
     const visBrukereMedAdressebeskyttelseEllerSkjermingStatus =
         useFeatureSelector()(VIS_MELDING_OM_BRUKERE_MED_ADRESSEBESKYTTELSE_ELLER_SKJERMING) &&
         props.oversiktType === OversiktType.enhetensOversikt &&
-        statusTall.adressebeskyttelseEllerSkjermingTotalt > 0;
+        props.statustall.utenBrukerinnsyn !== null &&
+        (props.statustall.utenBrukerinnsyn.ufordelteBrukere > 0 ||
+            props.statustall.utenBrukerinnsyn.venterPaSvarFraNAV > 0);
+
```

Comment:
bare helt på siden så kunne man kanskje kjørt no 
`const {medBrukerinnsyn, utenBrukerinnsyn} = props.statustall`
for å lesbarhet nedover i koden?